PennCourses/settings/base.py

```diff
@@ -56,6 +57,18 @@
     'review',
 ]
 
+#From labs-accounts
+AUTHENTICATION_BACKENDS = [
+    'accounts.backends.LabsUserBackend',
+    'django.contrib.auth.backends.ModelBackend',
```

Comment:
I think eventually we'll want to remove the ModelBackend, but it's a good idea rn to have a back-up if PennKeys fail for some reason

Reply:
Yeah I just added it because it was in the django-labs-accounts installation instructions but that makes sense